src/checks_util.cc

```diff
@@ -282,15 +289,19 @@ zetasql_base::StatusOr<VisitResult> RuleVisitor::defaultVisit(
   return VisitResult::VisitChildren(node);
 }
 
-std::vector<ParseToken> GetKeywords(absl::string_view sql) {
+std::vector<ParseToken> GetKeywords(absl::string_view sql, ErrorCode code) {
   ParseResumeLocation location = ParseResumeLocation::FromStringView(sql);
   std::vector<ParseToken> parse_tokens;
   std::vector<ParseToken> keywords;
 
   absl::Status status =
       GetParseTokens(ParseTokenOptions(), &location, &parse_tokens);
 
-  if (!status.ok()) return keywords;
+  if (!status.ok()) {
+    std::cout << "Tokenizer Failed for check [" << code
```

Comment:
I'd rephrase it other way round. The main message here is that we are skipping a check.
`std::cout << "Skipping check [" << code << "] due to tokenizer error: " << status.message()`

Reply:
Fixed